### DISCUSSION TITLE
The Six Cultures of play (link to blog)

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2372] [General Discussion]
This is an interesting blog post I found. It covers the old and new developments of 'how to roleplay' since the first rise of tabletop roleplaying in the 1970s. It's an interesting read also for MU-players/devs since we inherit a lot of our game mechanics and attitudes from tabletop play. 

https://retiredadventurer.blogspot.com/2021/04/six-cultures-of-play.html

It may also help to open one's eyes to the fact that there are other cultures/styles to explore - it's very easy to think that the style you grew up in or played the most is the one true way. This is very much true also for MU-games and their various communities.

----

I find myself very much steeped in the `Trad` culture since that's how I first learned roleplaying (and especially GMing). But the last few years I've tried several `Story Games` games too with good success. I own a few `Nordic LARP` style games but while they are interesting they feel like things you play in-person and I don't really know any players that would appreciate that kind of introspective play. 
I imagine my current preferred GM-style is somewhere between `Trad` and `Neo-Trad` today. I just enjoy making stories. :)

**Comment by alephate** [comment]
That was pretty interesting to read. I think I'm mostly in the OC range in terms of how I consider things, but my knowledge and experience in roleplaying is not very diverse, being primarily centered around MU games. :D 

I've been thinking about "immersion" and the "bleed" that it can cause in a setting where players are pitted against each other. Do you think it's possible to bleed like that and not ending up hating other members of the gaming community?

**Reply by Griatch** [reply]
I honestly think bleed in the sense that the Nordic-LARP guys mean it would require too much investment for an online game. I'm wary of people being too personal online. If one wanted that, the game should probably be an invite-only affair among people that know each other pretty well.

Competition in online games is of course nothing new though.

**Comment by taelspinner** [comment]
I've dipped my feet in both trad and OSR and enjoyed them. It really depends on what experience you're trying to create; these styles are tools that are aimed at replicating a certain theme. I wonder what category this person would slot "rules-lite" games under; probably OC?

**Reply by Griatch** [reply]
Most 'rules lite' games I know fall under 'story games' according to this classification. But compared to the really 'crunchy' rule systems out there, I'd imagine both OSR and OC could potentially be 'rules lite' depending on the system ...
